stylesheets/components/alert.css

```diff
@@ -0,0 +1,68 @@
+@import url("/stylesheets/main.css");
+
+body{
+    background-color: #f8f8ff;
```

Comment:
You can use var() to give background color.

Reply:
I've done as suggested , thankyou.